src/tanu/Assignment19.java

```diff
@@ -0,0 +1,42 @@
+package tanu;
+
+import java.util.Scanner;
+import java.lang.Character;
+
+class Assignment19 {
+	char letter;
+	int countupper = 0;
+	int countlower = 0;
+	int countdigit = 0;
```

Comment:
if this variable is common for other methods then we should write it at the instance level.